MAINTAINERS.md

```diff
@@ -67,27 +67,6 @@ You can then use it as
 checkout-pr estark37 example-fix
 ```
 
-### Merging pull requests into master
```

Comment:
This section, telling you to use the merge button, still seems valuable. We might even want to expand it to explain that if you use squash-and-merge, you need to delete the `(#XX)`.